### DISCUSSION TITLE
Youtube download button not appearing

### DISCUSSION THREAD

**Discussion by awaixjvd** [filecxx/FileCentipede#864] [General]
I just installed FC and also installed the chrome extension from chrome web store because the .crx file was blocked/greyed by chrome as it was installed from outside of store (i think).
I don't see any download bar/button on youtube videos. Am i missing something? 

Please guide.

**Comment by filecxx** [comment]
https://github.com/filecxx/PublicLists/discussions

**Reply by awaixjvd** [reply]
I really apologize and saying sorry for being noob here. I saw that link you gave but there is some link which i don't understand. Can you please guide as what shall i do to make it work.

I downloaded the extension from below link and i don't know what next shall i do to download from youtube.
https://chromewebstore.google.com/detail/file-centipede/ogfhebakocdbenmnkaajmckmbcnnafmm

Edit: also tried firefox and it doesn't work there too.

**Comment by filecxx** [comment]
Due to Chrome Web Store policies, this browser extension cannot directly support YouTube. However, it can integrate a third-party query interface to allow video file downloads.

When you click on a link shared by a user, a pop-up window will appear if you have the extension installed. This window will prompt you to add the interface. After clicking the confirm button and refreshing the YouTube page, you will see the download button appear.

**Reply by awaixjvd** [reply]
Again sorry for what i am writing. Please forgive not being able to get.

You said when i will click a link shared by user? what this means? i tried sharing the youtube video link to whatsapp and then clicked it and i didn't see any popup window to add the interface or no download button appearing. 
I have the chrome extension installed. It picks only audio file from youtube video but even if i try to download it, the file is empty.

**Reply by filecxx** [reply]
https://github.com/filecxx/PublicLists/discussions/7
https://github.com/filecxx/PublicLists/discussions/6
